src/compas/geometry/frame.py

```diff
@@ -24,6 +24,9 @@
 from .point import Point
 from .quaternion import Quaternion
 
+from scipy.spatial.transform import Rotation as R
+from scipy.spatial.transform import Slerp
```

Comment:
Since the implementation is now pure-python, please remove these ones, because they are causing the build to break.
```suggestion
```